core/kazoo_schemas/src/kz_json_schema_extensions.erl

```diff
@@ -115,3 +116,44 @@ validate_attachment_oauth_doc_id(Value, State) ->
             lager:debug("~s", [ErrorMsg]),
             jesse_error:handle_data_invalid('external_error', ErrorMsg, State)
     end.
+
+maybe_check_params_stability_level(Value, Schema, State) ->
+    case {kz_json:get_value(<<"properties">>, Schema),
+          kz_json:get_value(<<"id">>, Schema),
+          kz_json:is_json_object(Value)}of
```

Comment:
Drop the `}` to newline and `of` to a newline after that:

```erlang
case {xyz
     ,pdq
     ,abc
     }
of
```

See [here](https://github.com/2600hz/kazoo/blob/master/core/kazoo_endpoint/src/kz_endpoint.erl#L146-L150) for an example